frontend/src/components/card/LeaderboardCard.tsx

```diff
@@ -5,34 +5,60 @@ import { LowMarginText, SmallText } from '../core/Text';
 import PlayerIcon from './PlayerIcon';
 import { Color } from '../../api/Color';
 
-const Content = styled.div`
+type ContentStyleType = {
+  isCurrentPlayer: boolean,
+}
+
+const Content = styled.div<ContentStyleType>`
   display: inline-block;
   position: relative;
-  margin: 10px;
-  width: 7rem;
+  margin: 10px 5px;
+  width: 60px;
+  height: 80px;
+  padding: 10px 10px 5px 10px;
   
   &:hover {
     cursor: pointer;
   }
+  
+  ${({ theme, isCurrentPlayer }) => isCurrentPlayer && `
+    border-radius: 5px;
+    box-shadow: 0 -1px 5px rgba(0, 0, 0, 0.12);
+    background: ${theme.colors.white};
+  `};
+`;
+
+const CenteredScrollableContent = styled.div`
+  text-align: center;
+  overflow-x: scroll;
+
+  /* Hide scrollbar for Chrome, Safari and Opera */
+  &::-webkit-scrollbar {
+    display: none;
+  }
+  
+  /* Hide scrollbar for IE, Edge and Firefox */
+  -ms-overflow-style: none;  /* IE and Edge */
+  scrollbar-width: none;  /* Firefox */
 `;
 
 const HoverBar = styled.div`
-  // Center div above parent
+  // Center div over parent
   position: absolute;
+  z-index: 5;
   top: 0;
   left: 50%;
   transform: translate(-50%, 0);
   
-  width: 160px;
+  // width+padding, height+padding must match that of Content to perfect overlap
+  width: 70px;
+  min-height: 85px;
+  padding: 5px;
```

Comment:
Given this constraint, any thoughts on using a variable to match the Content and HoverBar components?

Reply:
Hm interesting, do you know how this would work? Like would it be a constant CSS variable at the top that you could edit to modify both at the same time?

Reply:
Yeah, you could add a constant `contentWidth` at the top, and then set the internal widths using `${contentWidth}px` (in addition to whatever additions or subtractions you want within the "{" and "}"). That way we can update this to change both components, so they should always be in sync.

Reply:
Just realized I totally forgot about this comment lmao - maybe I can implement it in the current open PR if you want to resurface it there?